### PR TITLE
[Docment][Feature] Refactor local parameter docment

### PR DESCRIPTION
**Pull request by QuakeWang**
ref: #9989

### CODE REVIEW

File: docs/docs/en/guide/parameter/local.md [hunk 1]
```diff
@@ -2,18 +2,27 @@
 
 ## Scope
 
-Parameters configured on the task definition page, the scope of this parameter is inside this task only. But if you configure according to [Refer to Parameter Context](context.md), it could pass to downstream tasks.
+Local parameters are used only for a single task node. Parameters configured on the task definition page, the scope of this parameter is inside this task only. But if you configure according to [Refer to Parameter Context](context.md), it could pass to downstream tasks.
```
Comment (SbloodyS) [inline docs/docs/en/guide/parameter/local.md]: I think the previous version's description is better. We should keep it as it is.

File: docs/docs/en/guide/parameter/local.md [hunk 2]
```diff
@@ -2,18 +2,27 @@
...
 
-<p align="center">
-     <img src="/img/supplement_local_parameter_en.png" width="80%" />
-</p>
+## Examples
 
-<p align="center">
-     <img src="/img/global_parameter_en.png" width="80%" />
-</p>
+This example uses local parameters to print the current date. Create a Shell task and write a script with the content 'echo ${dt}'. Click ** custom parameter ** in the configuration bar, and the configuration is as follows:
```
Comment (SbloodyS) [inline docs/docs/en/guide/parameter/local.md]: Do you think this is better? ```This example shows how to use local parameters to print the current date.```

File: docs/docs/zh/guide/parameter/local.md [hunk 3]
```diff
@@ -2,18 +2,27 @@
 
 ## 作用域
 
-在任务定义页面配置的参数，默认作用域仅限该任务，如果配置了[参数传递](context.md)则可将该参数作用到下游任务中。
+本地参数是只针对单个任务工作流节点的参数。在任务定义页面配置的参数，默认作用域仅限该任务，如果配置了[参数传递](context.md)则可将该参数作用到下游任务中。
```
Comment (SbloodyS) [inline docs/docs/zh/guide/parameter/local.md]: Same as english's.

File: docs/docs/zh/guide/parameter/local.md [hunk 4]
```diff
@@ -2,18 +2,27 @@
...
 
-<p align="center">
-   <img src="/img/supplement_local_parameter.png" width="80%" />
- </p>
+## 任务样例
 
-<p align="center">
-   <img src="/img/global_parameter.png" width="80%" />
-</p>
+该样例使用本地参数，打印输出当前日期。创建一个 Shell 任务，并编写脚本内容为 `echo ${dt}`。点击配置栏中的**自定义参数**，配置如下图所示：
```
Comment (SbloodyS) [inline docs/docs/zh/guide/parameter/local.md]: 本样例展示了如何使用本地参数.

File: docs/docs/en/guide/parameter/local.md (hunk 2 above)
Comment (QuakeWang) [reply docs/docs/en/guide/parameter/local.md]: > Do you think this is better? `This example shows how to use local parameters to print the current date.`

Great I will update it.

**Comment by zhongjiajie** [comment]
Do you finish your change now? @QuakeWang

**Comment by QuakeWang** [comment]
> Do you finish your change now? @QuakeWang

I have changed it. PTAL

**Comment by zhongjiajie** [comment]
I had restart the failed CI

File: docs/docs/en/guide/parameter/local.md [hunk 5]
```diff
@@ -6,14 +6,23 @@ Parameters configured on the task definition page, the scope of this parameter i
...
 
-<p align="center">
-     <img src="/img/supplement_local_parameter_en.png" width="80%" />
-</p>
+## Examples
 
-<p align="center">
-     <img src="/img/global_parameter_en.png" width="80%" />
-</p>
+This example shows how to use local parameters to print the current date. Create a Shell task and write a script with the content 'echo ${dt}'. Click ** custom parameter ** in the configuration bar, and the configuration is as follows:
```
Comment (zhongjiajie) [inline docs/docs/en/guide/parameter/local.md]:
```suggestion
This example shows how to use local parameters to print the current date. Create a Shell task and write a script with the content `echo ${dt}`. Click **custom parameter** in the configuration bar, and the configuration is as follows:
```

File: docs/docs/en/guide/parameter/local.md [hunk 6]
```diff
@@ -6,14 +6,23 @@ Parameters configured on the task definition page, the scope of this parameter i
...
-If you want to call the [built-in parameter](built-in.md) in the local parameters, fill in thevalue of built-in parameters in `value`. As in the above figure, `${biz_date}` and `${curdate}`.
\ No newline at end of file
+![local-parameter01](/img/new_ui/dev/parameter/local_parameter01.png)
+
+Parameters:
+
+-dt: indicates the parameter name
+-in: IN indicates that local parameters can only be used on the current node, and OUT indicates that local parameters can be transmitted to the downstream
+- DATE: indicates the DATE of the data type
+- $[YYYY-MM-DD] : indicates a built-in parameter derived from a user-defined format
```
Comment (zhongjiajie) [inline docs/docs/en/guide/parameter/local.md]:
```suggestion
- dt: indicates the parameter name
- in: IN indicates that local parameters can only be used on the current node, and OUT indicates that local parameters can be transmitted to the downstream
- DATE: indicates the DATE of the data type
- $[YYYY-MM-DD] : indicates a built-in parameter derived from a user-defined format
```

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**